Dockerfile

```diff
@@ -148,7 +149,7 @@ HEALTHCHECK --interval=20s --timeout=10s --retries=3 \
 # apache base build
 ###########################
 
-FROM php:8.0.14-apache-buster AS apache-base
+FROM php:8.1-apache-buster AS apache-php-ext-base
```

Comment:
out of curiosity: does renaming the base image change anything? might someone rely on that name? is it linked somewhere in the build process?